### PR TITLE
i18n

### PR DESCRIPTION
**Pull request by delphiactual**
- [x] `de`
 - [x] `it`
 - [x] `pt-br`
 - [x] `es`
 - [ ] `fr` (66%)
 - [ ] `ja` (26%) 

This is good to go whenever, the completion percentage here is just for reference.

### CODE REVIEW

File: app/views/en/support.html [hunk 1]
```diff
@@ -4,27 +4,27 @@
         <p>DIM is a product that is free to download and use. The source code for DIM is open sourced, and free to anyone to enhance. You will never see an ad in DIM. That is our commitment.</p>
         <p>We're all huge fans of Destiny. DIM came to life in those hours of gameplay because we felt that pain of transferring gear between characters and forgetting that one item, or trips down to the tower to unload your ROC strikes. DIM is the tool that will help you get more gametime.</p>
         <h2>How can I contribute?</h2>
-        <p>
+        <div>
             <dl>
-                <dt>Give DIM a Tip!</dt>
+<dt>Give DIM a Tip!</dt>
```
Comment (bhollis) [inline app/views/en/support.html]: What's with the lost indentation here?

Comment (delphiactual) [inline app/views/en/support.html]: thats how transifex outputs them

Comment (bhollis) [inline app/views/en/support.html]: Bummer

**Comment by bhollis** [comment]
I'll merge whenever you want.

**Comment by delphiactual** [comment]
its ready

**Pull request merged**